pulp_ansible/app/migrations/0025_deprecation.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 2.2.16 on 2020-10-12 17:15
+
+from django.db import migrations, models
+import django.db.models.deletion
+import django_lifecycle.mixins
+import uuid
+
+
+class Migration(migrations.Migration):
```

Comment:
There should be a step here to migrate the deprecation status over to `MutableCollectionMetadata` before the deprecated field is removed from the collections table.